apps/questions/templates/a4_candy_questions/question_module_detail.html

```diff
@@ -17,9 +13,10 @@ <h3>{{ module.name }}</h3>
                 <div class="mb-3">
                     {{ module.description | transform_collapsibles | richtext }}
                 </div>
-                <a class="btn btn--primary" href="{% url 'module-detail' module.project.organisation.slug module.slug %}">{% trans 'Go to event' %}</a>
+                {% if not module.future_phases %}
```

Comment:
@fuzzylogic2000 this seemed like a better choice as we don't repeat ourselves now

Reply:
Yes, you're right!